ServerClient/src/main/UserController.java

```diff
@@ -50,9 +50,9 @@ private void showAllUsersBooks() {
 
 	private void addBookToUsersList() {
 		ArrayList<Book> listBooks = getAllLibraryBooks();
-		if (listBooks.equals(null)) {
+		if (listBooks == null) {
 			return;
-		} else if (!(user.equals(null))) {
+		} else if (!(user == null)) {
```

Comment:
"else if" is unnecessary, use "if" instead
change !(user == null) to user != null